tests/service/connectors/test_queryconfig.py

```diff
@@ -623,27 +625,67 @@ def test_generate_query(self, policy, combined_traversal, connection_config_saas
 
         # static path with single query param
         config = SaaSQueryConfig(member, endpoints)
-        prepared_requests = config.generate_query({"query": ["customer-1@example.com"]}, policy)
+        prepared_requests = config.generate_query(
+            {"query": ["customer-1@example.com"]}, policy
+        )
         assert prepared_requests[0] == (
+            "GET",
             "/3.0/search-members",
             {"query": "customer-1@example.com"},
-            {}
+            None,
         )
 
         # static path with multiple query params with default values
         config = SaaSQueryConfig(conversations, endpoints)
-        prepared_requests = config.generate_query({"placeholder": ["customer-1@example.com"]}, policy)
+        prepared_requests = config.generate_query(
+            {"placeholder": ["customer-1@example.com"]}, policy
+        )
         assert prepared_requests[0] == (
+            "GET",
             "/3.0/conversations",
             {"count": 1000, "placeholder": "customer-1@example.com"},
```

Comment:
I missed this, what is this placeholder query param? I can't seem to find it in the docs

Reply:
The placeholder was used to create an artificial link to the `conversations` node. Since this endpoint doesn't have any dependencies on identity or references, this was the way I could get this node to be considered reachable without making changes to the traversal logic.

Ideally, it would be nice to have nodes like these (without dependencies) to be treated as root nodes and still be considered reachable.

Reply:
ah ok this would be a great thing to document in the saas config guides -